src/widget/about/aboutuser.h

```diff
@@ -23,7 +23,7 @@ class AboutUser : public QDialog
     ToxId toxId;
 
 private slots:
-    void onAcceptedClicked();
+    void onAcceptedClicked() const;
```

Comment:
I think this function changes the state too; even though it doesn't show a message box like the others, and so doesn't use the non-const `this` pointer, it does something similar to the other functions.
You shouldn't mark const everywhere there isn't a compiler error, only when it makes sense in the interface.

onAcceptClicked means somebody interracted with the widget and asked to save the new settings, so that counts as modifying things for me.